beacon-chain/state/state-native/getters_state.go

```diff
@@ -356,7 +397,7 @@ func (b *BeaconState) StateRootAtIndex(idx uint64) ([]byte, error) {
 // This assumes that a lock is already held on BeaconState.
 func (b *BeaconState) stateRootAtIndex(idx uint64) ([32]byte, error) {
```

Comment:
Comment that this function does not work with the MSV

Reply:
added a warning